__tests__/e2e/setup/.test-env

```diff
@@ -67,6 +67,12 @@ SESSION_SECRET=sandcrawler-138577
 SES_PORT=1025 
 SES_HOST=0.0.0.0
 
+# Remove these when SES migration is over (opengovsg/formsg-private#130)
+SES_PORT_US=1025
+SES_HOST_US=0.0.0.0
+SES_PORT_SG=1025
+SES_HOST_SG = 0.0.0
```

Comment:
Inconsistent (invalid?) syntax 
```suggestion
SES_HOST_SG=0.0.0.0
```

Reply:
OMG. this is why the tests are failing. thanks for catching this!!